docker/docker-compose.yml

```diff
@@ -2,9 +2,10 @@ version: '3.2'
 
 services:
     oms-frontend:
```

Comment:
you should remove this now, it's already built in the dockerfile

Reply:
@linuxbandit remove what?

Reply:
the block oms-frontend, we can only keep oms-frontend-nginx

Reply:
ah yeah, I planned to leave only oms-frontend and remove oms-frontend-nginx totally, but forgot to do it. Will update tonight.